draft-ietf-rats-architecture.md

```diff
@@ -360,6 +363,42 @@ are signed.  For example, values might have been generated at boot, and then
 used in claims as long as the signer can guarantee that they cannot have changed
 since boot.
 
+The term freshness is applied to both Evidence and Attestation Results in
+this document. Unfortunately, the term freshness is also a bit overloaded and
+in order to illustrate the application of the term in this document better,
+two characteristics that can be part of freshness are elaborated on below:
+recentness and uniqueness:
+
+Recentness:
+
+: a quality of Evidence that is based on the time that has passed between
+creation of the evidence by an Attester (typically by "sealing" a Claims set
+via signing) and the reception of that Evidence by a Verifier or (relaying
+party).
+
+: Evidence can be not recent enough, if the Evidence created from collected
+Claims is too old. A nonce supplied during a Challenge/Response handshake and
+a time-stamp based proof-of-recentness between time-synchronized peers are
+both usage scenarios in which recentness characteristics are provided to the
+Verifier.
+
+Uniqueness:
+
+: a quality of Evidence that is based on state of the Verifier that is
```

Comment:
I'm not sure why this should be Evidence specific?  (This also applies to Recentness above.)

Reply:
The entire text I wrote was Evidence-specific. Your comment relates to Dave's on lines 340+343 ("above"). I am hesitant to extend this beyond the scope of Evidence and Attestation Results (the latter is missing). Would the scope of Evidence + Attestation Results be okay?